kedro-org/kedro-viz
# Build out new Viz version panel

## Description

Resolves #1281.

## Development notes

I've augmented what we already had and built it out to the new design specs. 

## QA notes

Look at the branch as-is and you'll see the new version tag in the top left. That's the up-to-date state. To change the version of Viz in order to see the out-of-date state, change the version number in the `package/kedro_viz/__init__.py` to something before `5.3.0`.

We should probably think of a better color combo for the up-to-date state for the light theme. This is why: 

<img width="983" alt="image" src="https://user-images.githubusercontent.com/16869061/225743101-dadee066-a8b3-477b-9b85-dc46722d68b7.png">


## Checklist
